README-ruRU.md

```diff
@@ -2213,15 +2212,13 @@
   Устаревший комментарий гораздо хуже отсутствующего комментария.
   <sup>[[ссылка](#comment-upkeep)]</sup>
 
-<!--- @FIXME --->
 > Хороший код подобен хорошей шутке: он не нуждается в пояснениях. <br>
 > -- Рус Ольсен (Russ Olsen)
-> &mdash; old programmers maxim, through [Russ Olsen](http://eloquentruby.com/blog/2011/03/07/good-code-and-good-jokes/)
+> &mdash; афоризм старого программиста, отсюда [Russ Olsen](http://eloquentruby.com/blog/2011/03/07/good-code-and-good-jokes/)
 
-<!--- @FIXME --->
 * <a name="refactor-dont-comment"></a>
   Не пишите комментарии для объяснения плохого кода. Перепишите код, чтобы он
-  говорил сам за себя. ("Do or do not &mdash; there is no try." Yoda)
+  говорил сам за себя.
   <sup>[[ссылка](#refactor-dont-comment)]</sup>
```

Comment:
For example, original doc:
https://github.com/bbatsov/ruby-style-guide/blob/master/README.md#refactor-dont-comment

There is no quotation after, but in the Russian version there is a translated quotation below. I think it is correct to remove the Yoda's statement in parens, but keep the formalized quotation below.
![yoda-s double statement](https://cloud.githubusercontent.com/assets/3630353/19956627/d654fd62-a19f-11e6-8cbe-798318eb5ba3.png)